### ISSUE TITLE
Entities being ridden by players despawn at their tracking range

### ISSUE THREAD

**Issue by ghost** [PaperMC/Paper#4680]
entity-tracking-range:
      players: 64
      animals: 32
      monsters: 32
      misc: 48
      other: 64

### What behavior is expected:
a horse being ridden by a player should inherit the player's tracking range, i.e the horse should have a tracking range of a player

### What behavior is observed:
When you are >32 blocks from another player that is riding a horse, the horse despawns but the player doesn't. This creates a strange bug where the player appears to float in place at the location the horse despawned.

### Paper version:
git paper 244 1.16.3

**Comment by roosejt** [comment]
Hi dudeman123, I am currently looking into a more permanent solution for this. A temporary fix for you could be to set the server entity tracking flags of animals to equal players for now. This would slightly decrease performance, but may be negligible depending upon your server's hardware.

**Comment by Wyatt-James** [comment]
Is there any update on this? It behaves strangely on Spigot as well, causing the vehicle to have an extremely far tracking range. Maybe bug upstream?

**Comment by NoahvdAa** [comment]
This appears to have been fixed in 1.17, I can no longer reproduce this with these settings (but could in 1.16):
```yaml
    entity-tracking-range:
      players: 32
      animals: 16
      monsters: 16
      misc: 16
      other: 16
```

https://user-images.githubusercontent.com/44026893/149642307-26cc4441-4ed3-4a75-acfb-0191db3ab2b7.mp4
